bin/pidtree.js

```diff
@@ -3,19 +3,133 @@
 'use strict';
 
 var os = require('os');
+var pidtree = require('..');
 
-//
-// Change the default parent PID if running
-// under Windows.
-//
-let ppid = 1;
-if (process.platform === 'win32') {
-  ppid = 0;
+function help() {
+  var help = `  Usage
+    $ pidtree <ppid>
+
+  Options
+    --list                     To print the pids as a list.
+
+  Examples
+    $ pidtree
+    $ pidtree --list
+    $ pidtree 1
+    $ pidtree 1 --list
+`;
+  console.log(help);
+}
+
+function list(ppid) {
+  pidtree(ppid === undefined ? -1 : ppid, function(err, list) {
+    if (err) {
+      console.error(err.message);
+      return;
+    }
+    console.log(list.map(e => e.pid).join(os.EOL));
+  });
 }
 
-require('..')(process.argv[2] || ppid, (err, data) => {
-  if (err) {
-    return console.error(err);
+function tree(ppid) {
+  pidtree(ppid, function(err, list) {
+    if (err) {
+      console.error(err.message);
+      return;
+    }
+
+    var fathers = {}; // HasMap of fathers
```

Comment:
parents? Father is weird but I get it :smile:

Reply:
Thanks 🙈in italian is called `vettore dei padri` where `padri` = `fathers`.
But in english is called `vector of parents` actually.  😅